modules/saml/lib/SP/LogoutStore.php

```diff
@@ -365,6 +357,7 @@ public static function addSession($authId, $nameId, $sessionIndex, $expire)
         // serialize and anonymize the NameID
         // TODO: remove this conditional statement
         if (is_array($nameId)) {
+            /** @psalm-suppress UndefinedMethod */
```

Comment:
This is wrong. `NameID::fromArray()` exists in the 3.x branch of the SAML2 library, but it was deprecated, and removed in the 4.x branch. Our current `composer.json` allows both branches to be installed, so psalm was right reporting this. It is an actual bug that we need to fix.

Reply:
I disagree. Psalm runs on PHP7.3 with composer update, so using the 4.x branch of saml2